static/app/components/gridEditable/index.stories.tsx

```diff
@@ -280,4 +280,52 @@ export default Storybook.story('GridEditable', story => {
       </Fragment>
     );
   });
+
+  story('Enforcing Cell to fit Content', () => {
+    const newData = [
+      ...data,
+      {
+        name: 'Something very long',
+        category:
+          'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent fringilla ultricies turpis, quis lobortis leo varius ut. Maecenas venenatis purus a sodales facilisis.',
+      },
+    ] as ExampleDataItem[];
+    return (
+      <Fragment>
+        <p>
+          Passing
+          <Storybook.JSXProperty name="fitMaxContent" value={Boolean} /> will set the
+          width of the grid to fit around the content. By default this is set to false.
+        </p>
```

Comment:
It is a good idea to explain why/when this is useful and what cases it solves for. People are generally good at telling how things work, but they have a harder time understanding when to use it, or what the intent behind it was.